src/screens/Guide.js

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from 'react'
+import _ from 'lodash'
+import { Link } from 'react-router-dom'
+import PageTitle from 'components/PageTitle'
+import styled from 'styled-components'
+import { ListGroup, Alert } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+import { colors } from 'config'
+import ReactPlayer from 'react-player'
+import { useSelector } from 'react-redux'
+import { ppeGuidesData } from 'state/Response'
+
+const Guide = ({ match }) => {
+  const ppeGuides = useSelector(ppeGuidesData)
+  const { id } = match.params
+  const data = _.find(ppeGuides, ['id', parseInt(id, 0)])
```

Comment:
This logic could be moved to the selector so we selectPPEGuideById and pass in the id we want to use. I'll approve this now but would like to see this improved in future.